### PR TITLE
Remove "Generate Index" on hover for missing embeddings

### PR DESCRIPTION
**Pull request by toolmantim**
The "Generate Index" button is a bit misleading for dot com users, and for people who aren't site admins (as per discussion in #52206) so this just keeps it static as "Not Indexed". It also now just uses the standard link colours.

Before:

https://github.com/sourcegraph/sourcegraph/assets/153/d36e7922-b026-4094-b0bd-8e4401d30182

After:

https://github.com/sourcegraph/sourcegraph/assets/153/3c8882b3-f5f5-4aed-a1fb-ebdcf810d755

## Test plan

- Tested locally

### CODE REVIEW

File: client/cody-ui/src/chat/inputContext/ChatInputContext.module.css [hunk 1]
```diff
@@ -50,15 +50,4 @@
...
-a:hover .indexStatus {
-    display: none;
-}
-
-.indexStatus,
-a:hover .indexStatusOnHover {
-    display: inline;
-    color: var(--vscode-list-warningForeground, #fdf1cc);
-}
+}
```
Comment (abeatrix) [inline client/cody-ui/src/chat/inputContext/ChatInputContext.module.css:53]:
```suggestion
.indexStatus {
    display: inline;
    color: var(--vscode-list-warningForeground, #fdf1cc);
}
```
Should we keep the `color: var(--vscode-list-warningForeground, #fdf1cc);`? Removing this caused the status text to fallback using color for links, which is green on my end 👀 :
image

Comment (abeatrix) [inline client/cody-ui/src/chat/inputContext/ChatInputContext.module.css:53]:
```suggestion
.indexStatus {
    display: inline;
    color: var(--vscode-list-warningForeground, #fdf1cc);
}
```
Should we keep the `color: var(--vscode-list-warningForeground, #fdf1cc);`? Removing this caused the status text to fallback using color for links, which is green on my end 👀 :
image

File: client/cody-ui/src/chat/inputContext/ChatInputContext.tsx [hunk 2]
```diff
@@ -49,10 +49,7 @@ export const ChatInputContext: React.FunctionComponent<{
                 </h3>
             ) : contextStatus.supportsKeyword ? (
                 <h3 title={warning} className={classNames(styles.badge, styles.indexMissing)}>
-                    <a href="https://docs.sourcegraph.com/cody/explanations/code_graph_context">
-                        <span className={styles.indexStatus}>⚠ Not Indexed</span>
-                        <span className={styles.indexStatusOnHover}>Generate Index</span>
-                    </a>
+                    <a href="https://docs.sourcegraph.com/cody/explanations/code_graph_context">⚠ Not Indexed</a>
```
Comment (abeatrix) [inline client/cody-ui/src/chat/inputContext/ChatInputContext.tsx:52]:
```suggestion
                    <a href="https://docs.sourcegraph.com/cody/explanations/code_graph_context"><span className={styles.indexStatus}>⚠ Not Indexed</span></a>
```

**Review by abeatrix — COMMENTED** [review]
Thanks for the quick turnaround. left comments inline 😄

File: client/cody-ui/src/chat/inputContext/ChatInputContext.module.css [hunk 3]
```diff
@@ -52,13 +52,7 @@
...
-    display: none;
-}
-
-.indexStatus,
-a:hover .indexStatusOnHover {
+.indexStatus {
     display: inline;
-    color: var(--vscode-list-warningForeground, #fdf1cc);
-}
+    color: var(--vscode-list-warningForeground, inherit);
```
Comment (toolmantim) [inline client/cody-ui/src/chat/inputContext/ChatInputContext.module.css:57]: @abeatrix what was the purpose of this hard-coded value? Is that for web? One idea would be to make it `inherit` for that then?

Comment (abeatrix) [reply client/cody-ui/src/chat/inputContext/ChatInputContext.module.css:57]: Yea I think we added that value as a fallback for web.

**Review by abeatrix — APPROVED** [review]

**Pull request merged**